package.json

```diff
@@ -4,10 +4,23 @@
   "dependencies": {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement [previous reviewer's  suggestion](https://github.com/ouasamine/rails-webpack-demo/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519:~:text=react%2Ddom%20and-,babel%2Dloader,-.%20These%20are%20required) because you forgot install `babel-loader`. Please implement it 🙏🏼